namd/tests/library/016_mts_bias/test.in

```diff
@@ -7,7 +7,7 @@ colvar {
     name one
 
     outputAppliedForce on
-    timeStepFactor 0
+    timeStepFactor 4
```

Comment:
Why is the MTS test changed?

Reply:
0 should not be a legal value, so that needs to be fixed. Other than than that the physical trajectory should not change as long as the value divides 4 (the interval of the bias), so I picked 4 exactly